src/components/ActionList/ActionList.scss

```
@@ -137,6 +137,11 @@ $item-vertical-padding: ($item-min-height - line-height(body)) / 2;
   margin-left: spacing();
 }
 
+.ContentBlock,
+.ContentBlock span {
```

```suggestion
.ContentBlock,
.Prefix,
.Text,
.Content {
```

Targeting html elements can be fragile. Let's be explicit to make sure this doesn't break in the future

---

Ok so this broke it again :D I'll check what's happening but I'm guessing the class specificity &&|| the missing inner spanon .ContentBlock  is breaking flex or something

---

So actually adding .Prefix, .Text, .Content broke it since those already had defined flex display. Dropping those from the scss fixed it, and instead of using HTML tag span for the ContentBlock I added one more class `ContentBlockInner`